### PR TITLE
updating external file printing documentation to reflect changes to R…

### PR DESCRIPTION
**Pull request by Donald-Frederick**
…ingtail to follow RFC 7240 async http request pattern

### CODE REVIEW

File: ExternalFilePrinting.md [hunk 1]
```diff
@@ -18,9 +18,11 @@ Here's the flow for extending Ringtail's production printing process:
 
 1. The extension returns:
    - HTTP status code of 200 (OK) if the file is ready and returned in the body of the response.
-   - HTTP status code of 204 (No Content) if the file is not yet ready, indicating that Ringtail should retry the request at a later time.
-      - Use this response code to indicate that the request is being successfully processed but the file is not yet ready.
-      - Ringtail will retry file requests for approximately an hour. After that, Ringtail will mark the document as `Error Printing - External`.
+   - HTTP status code of 202 (Accepted) with a Location header set if the file is not ready yet, indicating that Ringtail should make a request to the specified location at a later time.  see [RFC 7240](https://tools.ietf.org/html/rfc7240#section-4.1)
```
Comment (Coldarn) [inline ExternalFilePrinting.md]: I would suggest mentioning this up top, perhaps right after the intro paragraph:

> To retrieve externally printed files, Ringtail supports the asynchronous HTTP request pattern per [RFC 7240](https://tools.ietf.org/html/rfc7240#section-4.1).

Then I'd change this line to:

> HTTP status code of 202 (Accepted) with a `Location` header set if the file is not ready yet, indicating that Ringtail should make a subsequent request to the specified location.

File: ExternalFilePrinting.md [hunk 2]
```diff
@@ -18,9 +18,11 @@ Here's the flow for extending Ringtail's production printing process:
...
+      - Use this response code to indicate that the request is being successfully processed and the file should be retrieved at a different location.
```
Comment (Coldarn) [inline ExternalFilePrinting.md]: The location doesn't HAVE to change on subsequent calls, I'd suggest this:
> Use this response code to indicate the request is being processed but the result is not yet ready.

File: ExternalFilePrinting.md [hunk 3]
```diff
@@ -18,9 +18,11 @@ Here's the flow for extending Ringtail's production printing process:
...
+      - Ringtail adds the same query string parameter `mainId` to the specified Location URL
+      - The specified location should follow the same rules for http responses.
```
Comment (Coldarn) [inline ExternalFilePrinting.md]: > If a `Location` header is used, the indicated query endpoint must follow the same rules for HTTP responses.

File: ExternalFilePrinting.md [hunk 4]
```diff
@@ -18,9 +18,11 @@ Here's the flow for extending Ringtail's production printing process:
 
 1. The extension returns:
    - HTTP status code of 200 (OK) if the file is ready and returned in the body of the response.
-   - HTTP status code of 204 (No Content) if the file is not yet ready, indicating that Ringtail should retry the request at a later time.
-      - Use this response code to indicate that the request is being successfully processed but the file is not yet ready.
-      - Ringtail will retry file requests for approximately an hour. After that, Ringtail will mark the document as `Error Printing - External`.
+   - HTTP status code of 202 (Accepted) with a Location header set if the file is not ready yet, indicating that Ringtail should make a request to the specified location at a later time.  see [RFC 7240](https://tools.ietf.org/html/rfc7240#section-4.1)
+      - Use this response code to indicate that the request is being successfully processed and the file should be retrieved at a different location.
+      - Ringtail adds the same query string parameter `mainId` to the specified Location URL
```
Comment (Coldarn) [inline ExternalFilePrinting.md]: Needs a period.

File: ExternalFilePrinting.md [hunk 5]
```diff
@@ -18,9 +18,11 @@ Here's the flow for extending Ringtail's production printing process:
...
+      - The specified location should follow the same rules for http responses.
+      - Ringtail will make a request to the specified location, and will rety file requests for approximately an hour.  After that, Ringtail will mark the document as `Error Printing - External`.
```
Comment (Donald-Frederick) [inline ExternalFilePrinting.md]: mispelled "retry"

**Pull request closed without merging**